docs/csharp/methods.md

```diff
@@ -137,7 +137,7 @@ The parameter's default value must be assigned by one of the following kinds of
 
 - A constant, such as a literal string or number.
 - An expression of the form `new ValType()`, where `ValType` is a value type. Note that this invokes the value type's implicit parameterless constructor, which is not an actual member of the type.
```

Comment:
I'm not sure if this statement will need to be tweaked with the new struct parameterless constructor feature.

@BillWagner Do you want a tracking issue for that?

Reply:
@Youssef1313 

Thanks for pointing this out. I've added a link to this comment in the note for the feature.

When each feature is merged into main, I'm spending time to build the full plans for the feature. This is captured on that note now.